support/cas-server-support-cookie/src/main/java/org/apereo/cas/web/WarningCookieRetrievingCookieGenerator.java

```diff
@@ -8,4 +8,8 @@
  * @since 4.2
  */
 public class WarningCookieRetrievingCookieGenerator extends CookieRetrievingCookieGenerator {
+
+    public WarningCookieRetrievingCookieGenerator(final String name, final String path, final int maxAge, final boolean secure) {
```

Comment:
Javadocs here please. (Failure to add will likely prevent CAS from publishing snapshots)